src/SprykerSdk/Shared/Transfer/ApplicationTransfer.php

```diff
@@ -1,7 +1,334 @@
 <?php
 
+/**
+ * (c) Spryker Systems GmbH copyright protected
+ */
+
 namespace SprykerSdk\Shared\Transfer;
 
-class ApplicationTransfer
+use InvalidArgumentException;
+
+class ApplicationTransfer extends AbstractTransfer
```

Comment:
I do not think that is a good idea
How are those updated? Manually?

Reply:
Yes basically what I did is just copied our Transfers and got rid of usage of Transfer`s module functionality.
From my perspective this is an easiest way to keep the code using DTO pattern.

Reply:
I dont think this is a good long term strategy..
It wont update with new fields.
Instead a different way of handling should be added IMO, but maybe that can be a follow up.